src/test/java/guitests/FilterTextFieldTest.java

```diff
@@ -40,15 +41,15 @@ private void testCompletions(FilterTextField field) {
         }
         // c[oun]t
         type("lo").push(KeyCode.TAB); // 'c' + 'lo' is a prefix of 'closed'
-        awaitCondition(() -> field.getText().equals("closedt"));
+        waitAndAssertEquals("closedt", field::getText);
     }
 
-    private void testSpaces(FilterTextField field) {
+    private void testSpaces_fieldNotEmpty_allowSpaces(FilterTextField field) {
```

Comment:
1. Method name should not start with `test`. The first part is the method or the feature being tested (i.e. SUT).
2. The convention is used for test methods (i.e. those with `@Test` annotations), not for utility methods.